Dockerfile.test.php7.debug

```diff
@@ -1,12 +1,12 @@
-FROM php:7.2
+FROM php:8.0.0
```

Comment:
This should not be saved. The CI does a `s/7.2/$versiontested/`

Reply:
@driusan but php 7.2, 7.3, 7.4 will all fail?

Reply:
@maltheism  Dockerfile.test.php7.debug doesn't run in CI by default . It only works on local VM.